packages/salesforcedx-vscode-core/package.json

```diff
@@ -28,6 +28,7 @@
     "@salesforce/apex-node": "0.0.9",
     "@salesforce/core": "2.9.0",
     "@salesforce/salesforcedx-utils-vscode": "49.12.0",
+    "@salesforce/schemas": "^1",
```

Comment:
This isn't technically needed since salesforce/core brings it in too, but I like to spell out dependencies that the code directly references.

Reply:
Yes, I agree on being explicit about that dependency